pkgs/by-name/km/kminion/update.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq nix-update common-updater-scripts
```

Comment:
```suggestion
#!nix-shell -i bash -p nix-update
```

Reply:
The dangers of copy-paste, nice catch :p